### DISCUSSION TITLE
被关闭的话题错误

### DISCUSSION THREAD

**Discussion by purainity** [Coolapk-Fan/wiki#8] [General]
在 [已被关闭、禁止访问、暂不可用的话题](https://github.com/Coolapk-Fan/wiki/wiki/%E5%8F%82%E4%B8%8E%E8%AF%9D%E9%A2%98%E8%AE%A8%E8%AE%BA%E7%AD%89%E7%BA%A7%E8%A6%81%E6%B1%82&%E9%99%90%E5%88%B6#%E5%B7%B2%E8%A2%AB%E5%85%B3%E9%97%AD%E7%A6%81%E6%AD%A2%E8%AE%BF%E9%97%AE%E6%9A%82%E4%B8%8D%E5%8F%AF%E7%94%A8%E7%9A%84%E8%AF%9D%E9%A2%98) 中，实测 #云流量# #蓝奏云# #欢太商城# #丰富黑名单# 等话题可正常访问。
![IMG_20230903_151513](https://github.com/Coolapk-Fan/wiki/assets/126877770/33231038-ba99-4b3a-92a0-3668962b3607)
![Screenshot_2023-09-03-15-07-37-602_com coolapk market](https://github.com/Coolapk-Fan/wiki/assets/126877770/f50a18fa-80a9-47a5-9f7d-179d6e3fd665)
![Screenshot_2023-09-03-15-07-45-072_com coolapk market](https://github.com/Coolapk-Fan/wiki/assets/126877770/1cb85c87-48d5-42ad-b649-bf18d07e69b5)
![Screenshot_2023-09-03-15-08-03-741_com coolapk market](https://github.com/Coolapk-Fan/wiki/assets/126877770/7329a19e-2b84-448e-a066-5ed546b9b907)
![Screenshot_2023-09-03-15-08-15-595_com coolapk market](https://github.com/Coolapk-Fan/wiki/assets/126877770/26d58046-fc31-4118-b4ee-0e91e924cb6c)

**Comment by Coolapk-Fan** [comment]
谢谢反馈，已将其移至 [曾经被关闭、禁止访问的话题](https://github.com/Coolapk-Fan/wiki/wiki/%E5%8F%82%E4%B8%8E%E8%AF%9D%E9%A2%98%E8%AE%A8%E8%AE%BA%E7%AD%89%E7%BA%A7%E8%A6%81%E6%B1%82&%E9%99%90%E5%88%B6#%E6%9B%BE%E7%BB%8F%E8%A2%AB%E5%85%B3%E9%97%AD%E7%A6%81%E6%AD%A2%E8%AE%BF%E9%97%AE%E7%9A%84%E8%AF%9D%E9%A2%98)
已开放Wiki编辑权限，若有内容修改或补充需求，可点击相关页面右上角“edit”按钮进行编辑
